dat/missions/flf/empa/flf_empa01.lua

```diff
@@ -0,0 +1,136 @@
+
+   text[8] = [[He stands and holds out his hand, which you shake. "It's been a pleasure to talk to you, %s. I hope to see you again in the future." He then leaves the bar. Time to report back to the FLF base!]]
+
+   title[9] = "A New Beginning"
+   text[9] = [[Cheryl waits for you as you land. "I trust your mission went well?" You confirm that, yes, you have spoken with Commander Petridis about the FLF's intentions. "Excellent," she remarks. She then gives you a firm handshake. "Congratulations, %s. You are now the official Empire ambassador of the FLF. Your payment for this first mission will be in your account shortly. Please check back later for your next mission." Cheryl then departs.]]
+
+   misn_title = "Empire Negotiations"
+   misn_desc = "You are to meet with Empire Commander Petridis and deliver a message that the FLF wishes to establish friendly relations with the Empire."
+   misn_reward = "A chance for a new beginning"
+
+   npc_name = "A young woman"
+   npc_desc = "This woman appears to be by herself, which is somewhat unusual for an FLF member. She regularly glances in your direction."
+
+   emp_name = "An Empire official"
+   emp_desc = "There is an Empire official sitting by himself, enjoying a drink. He seems to take special interest in you."
+
+   osd_title   = "Empire Negotiations"
+   osd_desc    = {}
+   osd_desc[1] = "Go to %s in the %s system and talk to Commander Petridis"
+   osd_desc[2] = "Return to FLF base and report back to Cheryl"
+   osd_desc["__save"] = true
+end
+
+
+function create ()
+   -- Note: this mission does not make any system claims.
+   missys = system.get( "Raelid" )
+   misplanet = planet.get( "Marius Station" )
+
+   credits = 100000
+   reputation = 20
+   emp_reputation = 5
+
+   misn.setNPC( npc_name, "neutral/miner2" )
+   misn.setDesc( npc_desc )
+end
+
+
+function accept ()
+   tk.msg( title[1], text[1]:format( player.name() ) )
+   tk.msg( title[1], text[2] )
+   if tk.yesno( title[1], text[3] ) then
+      tk.msg( title[1], text[4]:format( misplanet:name(), missys:name() ) )
+
+      misn.accept()
+
+      osd_desc[1] = osd_desc[1]:format( misplanet:name(), missys:name() )
+      misn.osdCreate( osd_title, osd_desc )
+      misn.setTitle( misn_title )
+      misn.setDesc( misn_desc )
+      marker = misn.markerAdd( missys, "plot" )
+      misn.setReward( misn_reward )
+
+      job_done = false
+
+      hook.land( "land" )
+   else
+   end
+end
+
+
+function land ()
+   if job_done and planet.cur():faction():name() == "FLF" then
+      tk.msg( title[9], text[9]:format( player.name() ) )
+      player.pay( credits )
+      var.push( "_fcap_flf", math.max( 60, var.peek( "_fcap_flf" ) ) )
```

Comment:
Instead of directly manipulating these variables in missions, I think it would be better to try to wrap it as done for empire in 'dat/missions/empire/common.lua'. If we ever change the mechanics or whatever it'll be cleaner and easier to handle, instead of directly manipulating obscure variables.